packages/atomic/src/pages/v1.html

```diff
@@ -0,0 +1,202 @@
+
+      atomic-search-interface {
+        grid-template-columns: 1fr minmax(150px, 325px) minmax(500px, 925px) 1fr;
+        grid-template-rows: 22px 50px 22px 1fr;
+        column-gap: 20px;
+        display: grid;
+        margin: 0 auto;
+      }
+
+      .header {
+        background-color: var(--atomic-background-variant);
+        grid-column: 1 / span 4;
+        grid-row: 1 / span 3;
+      }
+
+      atomic-search-box {
+        grid-column: 3 / auto;
+        grid-row: 2 / auto;
+        max-width: 650px;
+      }
+
+      atomic-facet-manager {
+        grid-column: 2 / auto;
+        grid-row: 4 / auto;
+        margin-top: 15px;
+      }
+
+      .results-container {
+        grid-column: 3 / auto;
+        grid-row: 4 / auto;
+        margin-top: 15px;
+      }
+
+      .side-by-side-container {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+      }
+
+      .footer-item {
+        margin: 30px 0;
+      }
+
+      atomic-facet-manager {
+        padding-top: 13px;
+      }
+
+      atomic-sort-dropdown {
+        margin-left: 15px;
+      }
+
+      atomic-result-list {
+        display: block;
+        margin-top: 10px;
+      }
+
+      atomic-breadcrumb-manager {
+        margin: 10px 0;
+      }
+
+      @media only screen and (max-width: 1023px) {
+        atomic-search-interface {
+          grid-template-rows: 16px 50px 16px auto 1fr;
+          grid-template-columns: minmax(0, 100%);
+        }
+
+        atomic-facet-manager,
+        .header,
+        atomic-search-box,
+        .results-container {
+          grid-column: 1 / auto;
+        }
+
+        atomic-breadcrumb-manager,
+        atomic-query-summary,
+        atomic-no-results,
+        atomic-did-you-mean,
+        atomic-query-error,
+        atomic-sort-dropdown,
+        atomic-facet-manager {
+          padding: 0 15px;
+        }
+
+        atomic-atomic-breadcrumb-manager,
+        atomic-query-summary,
+        atomic-no-results,
+        atomic-query-error,
+        atomic-sort-dropdown {
+          margin-bottom: 10px;
+        }
+
+        atomic-facet-manager {
+          grid-row: 4 / auto;
+          display: flex;
+          flex-wrap: wrap;
+        }
+
+        .results-container {
+          grid-row: 5 / auto;
+          margin-top: 0;
+        }
+
+        atomic-facet-v1 {
+          margin: 0 10px 0 0;
+        }
+
+        atomic-search-box {
+          justify-self: center;
+          min-width: 80%;
+          max-width: 80%;
+        }
+      }
+
+      @media only screen and (max-width: 640px) {
+        .side-by-side-container {
+          flex-direction: column;
+          align-items: flex-start;
+        }
+      }
+    </style>
+  </head>
+
+  <body>
+    <atomic-search-interface id="search">
+      <div class="header"></div>
+      <atomic-search-box></atomic-search-box>
+      <atomic-facet-manager>
+        <atomic-facet-v1 field="author" label="Authors"></atomic-facet-v1>
```

Comment:
New facets will have `-v1` appended to the tags, then old components will be replaced